components/toggle.tsx

```diff
@@ -2,33 +2,36 @@
 // See LICENSE.txt for license information.
 
 import React from 'react';
-import PropTypes from 'prop-types';
 
-export default function Toggle({onToggle, toggled, onText, offText, disabled}) {
+type Props = {
+    onToggle: (event: React.MouseEvent<HTMLButtonElement, MouseEvent>) => void;
+    toggled: boolean,
```

Comment:
`toggled`, `onText` and `offText` were optional before, why now are mandatory?

Reply:
Fixed